source/_posts/2021-12-23-the-open-home.markdown

```diff
@@ -0,0 +1,48 @@
+---
+title: "The Open Home"
+description: "Our vision for the smart home is one that is "
+date: 2021-12-23 00:01:00
+date_formatted: "December 23, 2021"
+author: Paulus Schoutsen
+author_twitter: balloob
+categories: Video
+og_image: /images/blog/2021-12-open-home/social.png
+---
+
+<img src='/images/blog/2021-12-open-home/header.png' style='box-shadow:none;border:0'>
+
+The Open Home is our vision for the smart home. It defines the values that we put at the heart of every decision we make at Home Assistant. It's woven into our architecture, licensing, community and everything else.
+
+The Open Home is about **privacy**, **choice** and **durability**.
+
+_If you prefer video, skip [to the end](#state-of-the-open-home)._
+
+## Privacy
+
+Your home should be your safe space. A place where you can be your true self without having to bother about what the world thinks of you. A place where you don't need to act differently to avoid an algorithm categorizing your behavior.
+
+**Privacy** for the Open Home means that devices need to work locally. You shouldn't need a cloud connection just to turn on a light bulb or change the thermostat.
+
+It is okay for a product to offer a cloud connection, but it should be opt-in.
+
+## Choice
+
+Devices in your home gather data about themselves and their surroundings. Your data. Vendors shouldn't be able to limit access to your data or limit the interoperability of their products with the rest of your smart home.
+
+**Choice** for the Open Home means that products need to make the gathered data available to the user through APIs. This avoids vendor lock-in and allows users to mix systems as they see fit.
+
+## Durability
+
+If there is one thing that technology firms are very good at, it is launching new products. However, maintaining the products and making sure they keep working is an afterthought for most. The result is that vendors will decide to turn off the support for your device, crippling the features or even prevent it from working at all. You will need to buy something new.
+
+As we install more and and more devices in our home, durability is becoming more and more important. We shouldn't have to buy everything new every couple of years because the manufacturer decided to move on and create something newer.
```

Comment:
```suggestion
As we install more and more devices in our home, durability is becoming more and more important. We shouldn't have to buy everything new every couple of years because the manufacturer decided to move on and create something newer.
```